### PR TITLE
Initial pull

### PR DESCRIPTION
**Pull request by beck**
ping @stefanor - this thing has been immensely helpful.

Can you please give me a review?

And if you're willing to install it and try it and find some use in it, then I'll introduce it to everyone else.

### CODE REVIEW

**Comment by beck** [comment]
:bell: Now with tests! :fireworks:

**Comment by beck** [comment]
@stefanor just a tiny review?

File: merge.sh [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+# if additional options are ever added will need to use getopts
+silent=false
+if [ ${1:-loud} = '-s' ]
+then
+  silent=true
+  shift
+fi
+
+if [[ -z ${1:-} ]] || [[ -z ${2:-} ]] || [[ -z ${3:-} ]]
```
Comment (stefanor) [inline merge.sh]: You want $#? Also [[ is a bashism

File: merge.sh [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+
+function verify_msgcat {
+  if ! $(msgcat --version > /dev/null 2>&1)
+  then
+    log
+    log "ERROR in git-po-merge: msgcat is not found."
+    log "  Installing gettext should include msgcat."
+    log "  Falling back to three way merge."
+    log
```
Comment (stefanor) [inline merge.sh]: You should write errors to stderr

**Pull request merged**